src/runtime_src/core/pcie/driver/linux/xocl/xocl_xgq_plat.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2016-2021 Xilinx, Inc. All rights reserved.
+ *
+ * Authors: davidzha@Xilinx.com
```

Comment:
Might not be best to put your name here.

Reply:
fixed